chart/values.yaml

```diff
@@ -3615,7 +3614,6 @@ externalRedis:
   tlsEnabled: false
   tlsCA: ""
   existingSecret: ""
-  existingSecretPasswordKey: ""
```

Comment:
Question, why don't we allow to choose the key name in the client's provided secret?

Reply:
mmm I cannot remember why I removed this 🤔  Maybe because we are using the `existingSecret` approach for this?

Reply:
But this is the key to look for in the Secret object in :k8s:.

If we currently don't use it, we should implement it